src/main/java/spireMapOverhaul/zones/smithsFolly/events/RansackedForge.java

```diff
@@ -0,0 +1,162 @@
+                        AbstractDungeon.combatRewardScreen.open();
+                        AbstractDungeon.getCurrRoom().rewardPopOutTimer = 0.0F;
+
+                        AbstractDungeon.combatRewardScreen.rewards.removeIf(reward -> reward.type == RewardItem.RewardType.CARD);
+
+                        // Update event text and options
+                        this.imageEventText.updateBodyText(DESCRIPTIONS[2]);
+                        this.imageEventText.updateDialogOption(0, OPTIONS[3]); // Text for closing the event
+                        this.imageEventText.clearRemainingOptions();
+                        this.screen = CurScreen.COMPLETE;
+                        break;
+
+                    case 2:
+                        // Leave
+                        this.imageEventText.updateBodyText(DESCRIPTIONS[3]);
+                        break;
+                }
+                this.imageEventText.updateDialogOption(0, OPTIONS[5]);
+                this.imageEventText.clearRemainingOptions();
+                this.screen = CurScreen.COMPLETE;
+                break;
+            case COMPLETE:
+                openMap();
+                break;
+        }
+    }
+
+    private enum CurScreen {
+        INTRO,
+        COMPLETE
+    }
+    private void handleSearingBlowOption() {
+        if (AbstractDungeon.player.masterDeck.findCardById(SearingBlow.ID) != null) {
+            // Upgrade Searing Blow four times
+            upgradeSearingBlow(4);
+            this.imageEventText.updateBodyText(DESCRIPTIONS[1]);
+        } else if (AbstractDungeon.actNum == 3 && hasStrike) {
+            // Replace all Strikes with Searing Blow+4
+            replaceStrikesWithSearingBlow();
+            this.imageEventText.updateBodyText(DESCRIPTIONS[4]);
+        } else {
+            // Obtain Searing Blow+4
+            AbstractCard searingBlow = CardLibrary.getCard(SearingBlow.ID).makeCopy();
+            searingBlow.upgrade();
+            searingBlow.upgrade();
+            searingBlow.upgrade();
+            searingBlow.upgrade();
+            AbstractDungeon.effectList.add(new ShowCardAndObtainEffect(searingBlow, (float)Settings.WIDTH / 2.0F, (float)Settings.HEIGHT / 2.0F));
+            this.imageEventText.updateBodyText(DESCRIPTIONS[5]);
+        }
+    }
+
+    private void upgradeSearingBlow(int times) {
+        for (AbstractCard card : AbstractDungeon.player.masterDeck.group) {
+            if (card.cardID.equals(SearingBlow.ID)) {
+                for (int i = 0; i < times; i++) {
+                    card.upgrade();
+                }
+                // Show visual effects for each upgrade
+                AbstractDungeon.topLevelEffects.add(new ShowCardBrieflyEffect(card.makeStatEquivalentCopy()));
+                AbstractDungeon.topLevelEffects.add(new UpgradeShineEffect((float)Settings.WIDTH / 2.0F, (float)Settings.HEIGHT / 2.0F));
+            }
+        }
+    }
+
+
+    private void replaceStrikesWithSearingBlow() {
```

Comment:
This might be too crazy. Removing two strikes and giving one searing blow+4 or removing all strikes and replacing them with unupgraded searing blows is probably better.

Reply:
Keep in mind this outcome can only occur in Act 3. Most players will probably have removed most of their Strikes by then, and even if they kept all the Strikes, I don't think a few Searing Blow+4 cards will drastically shift the run (2 cost, 34 damage. Carnage+ for comparison is 2 cost, 28 damage, not too far off). It can MAYBE be a desperate last minute damage solution if you don't have one already or it's just a good pick if you have Snecko Eye but I truly don't think it's that overpowered.

Reply:
I can see your point, but I still think people tend to have 2-3 Strikes in their deck by the end of the run on average. That's still replacing the worst attack in the game with cards that are better than an upgraded rare on Ironclad. That's a very strong upside, I fear to think what degeneracy Silent or Watcher could get up to with it. 

Maybe a compromise, upgrade them twice, that way it's 21 damage, still crazy strong but at least not win-con strong.

Reply:
I think 2 cost 21 damage cards towards the end of the run is pretty bad, even if they replace Strikes. Would you settle for Searing Blow+3 instead?

Reply:
It's not great but it's a big damage attack, characters that aren't balanced around having that would salivate. You won#t find a higher damage attack on watcher, it's probably the best attack on her and Silent still.

And it's certainly better than a strike would ever be.

Reply:
Ah, I didn't even think about what you could do with Watcher's stances. I block that character out of my mind.

Alright, I decided to make all the options +2 rather than +4 so they're uniform. That shouldn't be too powerful.